tests/integration/test_application_variables.py

```diff
@@ -16,22 +17,26 @@ def tearDown(self):
         settings.EQ_GOOGLE_TAG_MANAGER_AUTH = None
 
     def test_google_analytics_code_and_credentials_are_present(self):
-        self.launchSurvey("test_feedback")
+        self.launchSurvey("test_feedback", roles=["dumper"])
+        self.get("/dump/debug")
+        actual = json_loads(self.getResponseData())
         self._client.set_cookie(
             "localhost", key="ons_cookie_policy", value="'usage':true"
         )
         self.get("/questionnaire/feedback/")
         self.assertStatusOK()
         self.assertInHead("gtm.start")
         self.assertInHead(
-            'dataLayer = [{"form_type": "H", "survey_id": "0", "title": "Feedback test schema"}]'
+            f'dataLayer = [{{"tx_id": "{actual["METADATA"]["tx_id"]}"}}, {{"form_type": "H", "survey_id": "0", "title": "Feedback test schema"}}]'
```

Comment:
I thought we said it should be part of the already existing mapping?. Happy to keep as is, if it makes no difference to Ian and might be better as one is schema data.

Reply:
Ian said no difference and he is happy if it's this way, I prefer this approach as it's less "invasive" and we don't reassign existing var of the parent class,  just extend it.